frontend/src/bundles/common/icons/icon-name.ts

```diff
@@ -1,6 +1,16 @@
-import { faEllipsisVertical, faPen } from '@fortawesome/free-solid-svg-icons';
+import { ArrowLeftIcon, ArrowRightIcon } from '@chakra-ui/icons';
+import {
+    faEllipsisVertical,
+    faHouse,
+    faPen,
+    faRightFromBracket,
+} from '@fortawesome/free-solid-svg-icons';
 
 const IconName = {
+    HOME: faHouse,
+    LOG_OUT: faRightFromBracket,
+    ARROW_LEFT: ArrowLeftIcon,
+    ARROW_RIGHT: ArrowRightIcon,
     OPTIONS_VERTICAL: faEllipsisVertical,
     PEN: faPen,
 } as const;
```

Comment:
It would be better to create a mapper over FA icons that would unify ChakraUI and FA Icons. This way you will have one usage interface. You can convert FA icons to the way so they could also be used as `<Icon as={IconName.HOME} />`

Reply:
We already discussed a lot about this, we agreed to use it like this. I think Nikita agreed too